gate-web/src/main/groovy/com/netflix/spinnaker/gate/controllers/ManagedController.java

```diff
@@ -324,4 +331,13 @@ void deleteVeto(
       @PathVariable("version") String version) {
     keelService.deleteVeto(application, targetEnvironment, reference, version);
   }
+
+  @PostMapping(
+      path = "/notifications/callbacks/{source}/v2",
```

Comment:
Since this is mapped under `/managed`, I don't think you need the `/v2` here, right?

Reply:
true, will remove it!